revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
+        }
+
+        .internal-graphics-container {
+            height: 220px;
+            width: 220px;
+            position: relative;
+            left: 50%;
+            top: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+            position: absolute;
+        }
+
+        .project-badge-line, .project-badge-circle {
+          fill: none;
+          stroke-width: 20px;
+        }
+
+        .project-badge-line {
+          stroke: $revolv-blue;
+        }
+
+        .project-badge-circle {
+          stroke: white;
+        }
+
+        .text-container {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+            height: 220px;
+            width: 220px;
+
+            .percentage-text {
+                font-size: 3rem;
+                color: #064f7e;
+                font-weight: 500;
+                position: absolute;
+
+                left: 50%;
+                top: 40%;
+                -webkit-transform: translate(-50%, -50%);
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+            }
+
+            .funded {
+                font-size: 1.5rem;
+                color: #064f7e;
+                font-weight: 800;
+                position: absolute;
+
+                left: 50%;
+                top: 60%;
+                -webkit-transform: translate(-50%, -50%);
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+            }
+        }
+    }
+}
+
+.about {
+    margin-top: 200px;
+
+    .header-text {
+        p {
+            font-weight: 800;
+            color: black;
+            font-size: 1.8rem;
+        }
+    }
+    .gray-bar {
+        background-color: #999999;
+        height: 1px;
+        width: 100%;
+    }
+
+    .categories {
+        max-width: 100%;
+        height: 50px;
+
+
+        .category-text {
+            display: inline-block;
+            color: #888888;
+            margin-left: 15px;
+            margin-right: 10px;
+            height: 30px;
+
+            .category-text-inside {
+                line-height: 30px;
+                font-size: 0.8rem;
+            }
+        }
+
+        .category-holder {
+            display: inline-block;
+            border-color: #888888;
+            border-style: solid;
+            border-width: 1px;
+            width: 15%;
+            height: 30px;
+            margin-top: 10px;
+            margin-right: 5px;
+            margin-left: 5px;
+
+            .category-label {
+                color: #888888;
+                text-align: center;
+                line-height: 30px;
+            }
+        }
+    }
+
+    .about-text {
```

Comment:
I think in the mockup we had the text here as dark gray rather than black? Black is fairly jarring on this page I think